coffee-backend/build.gradle.kts

```diff
@@ -43,6 +43,7 @@ dependencies {
 	implementation("com.google.code.gson:gson:2.8.5") //Gson
 
 	implementation("org.jetbrains.kotlin:kotlin-noarg")
+	implementation("org.projectlombok:lombok")
```

Comment:
Do not use Lombok with Kotlin. It is a massive clusterfuck used in Java to make it less verbose, more like Kotlin 🦀

Reply:
Without Lombok we get StackOverflowError :( 

If you have a way of correctly mapping the ManyToMany User Friends entity that works in Kotlin, please give it to me. As most things have been tried

Reply:
https://github.com/santi/coffee-with-me/pull/10 should do the trick